app/components/specific/map/RaceMap.js

```diff
@@ -61,16 +70,24 @@ export default class RaceMap extends Component {
     const { race, clubMembers } = this.props;
 
     const sampleRegion = {
-      latitude: race.path.coordinates[0].latitude,
-      longitude: race.path.coordinates[0].longitude,
+      latitude: race.path.edges[0].src.latitude,
+      longitude: race.path.edges[0].src.longitude,
       latitudeDelta: LATITUDE_DELTA,
       longitudeDelta: LONGITUDE_DELTA
     };
 
+    const raceData = race.path.edges.reduce(
+      (locations, edge) => [
```

Comment:
last point is missing

Reply:
It was just to test how good reviewer you were! 👍